laborator/content/interactiune-c-assembly/08-include-fix/hello.c

```diff
@@ -0,0 +1,15 @@
+static void hi(void)
```

Comment:
```suggestion
// SPDX-License-Identifier: BSD-3-Clause

static void hi(void)
```

Same for all `.c` files.